GccBoardProject1/src/loraDriver.c

```diff
@@ -0,0 +1,285 @@
+/*
+ * loraDriver.c
+ *
+ * Created: 3/24/2017 12:08:18 PM
+ *  Author: mazacar8
+ */
+
+#include "loraDriver.h"
+#include <stdio.h>
+#include <asf.h>
+#include "saml21j18b.h"
+
+uint8_t reset_N = PIN_PB00;
+uint16_t rx_offset = 0;
+
+/**
+ * @brief      Initializes UART for the RN2903 and debug interface
+ *
+ */
+void init_uart() {
+	struct usart_config config_usart;
+	struct usart_config config_lora;
+	
+	//Debug Interface configuration
+	usart_get_config_defaults(&config_usart);
+	config_usart.baudrate    = DEBUG_BAUD;
+	config_usart.mux_setting = EDBG_CDC_SERCOM_MUX_SETTING;
+	config_usart.pinmux_pad0 = EDBG_CDC_SERCOM_PINMUX_PAD0;
+	config_usart.pinmux_pad1 = EDBG_CDC_SERCOM_PINMUX_PAD1;
+	config_usart.pinmux_pad2 = EDBG_CDC_SERCOM_PINMUX_PAD2;
+	config_usart.pinmux_pad3 = EDBG_CDC_SERCOM_PINMUX_PAD3;
```

Comment:
All of the serial debug stuff should probably be wrapped inside of `#IFDEF`'s. We don't want to be writing debug output if not necessary.